datafusion/core/src/lib.rs

```diff
@@ -52,11 +52,11 @@
 //!    .to_string();
 //!
 //! let expected = vec![
-//!     "+---+--------------------------+",
-//!     "| a | MIN(tests/example.csv.b) |",
-//!     "+---+--------------------------+",
-//!     "| 1 | 2                        |",
-//!     "+---+--------------------------+"
+//!     "+---+----------------+",
+//!     "| a | MIN(?table?.b) |",
```

Comment:
I'm not entirely sure what change caused this... I wonder if we aren't running doctests in CI :thinking:

Reply:
it seems ok to me. I am not sure if the `use the filename as table name` function of csv was intentional (like maybe it is some sort of dataframe compatibility)?

cc @andygrove